app-tasks/rf/src/rf/uploads/geotiff/create_thumbnails.py

```diff
@@ -0,0 +1,137 @@
+"""Creates thumbnails for geotiff imagery
+
+Depends on having geotiff file already downloaded
+"""
+import os
+import subprocess
+
+import boto3
+
+from rf.models import Thumbnail
+
+from .io import get_geotiff_dimensions
+
+def create_thumbnails(tif_path, scene_id, organization_id):
+    """Creates thumbnails based on a geotiff path and scene
+
+    Args:
+        tif_path (str): path of local tiff file
+        scene_id (str): uuid of the scene
+
+    Returns:
+        List[Thumbnail]
+    """
+
+    r_tif_path = 'resampled_{}'.format(tif_path)
```

Comment:
I did things slightly differently this time and realized that this won't work with absolute paths, which is what it will be passed from NamedTemporaryFile. I think this needs to become something like `os.path.join(os.path.dirname(tif_path), 'resampled_{}'.format(os.path.basename(tif_path)))`, ugly as that is.